plugin/src/testFixtures/kotlin/io/qalipsis/gradle/SSETestServer.kt

```diff
@@ -0,0 +1,126 @@
+package io.qalipsis.gradle
+
+import io.ktor.http.*
+import io.ktor.server.application.install
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import io.ktor.server.response.respondTextWriter
+import io.ktor.server.response.respondText
+import io.ktor.server.response.cacheControl
+import io.ktor.server.routing.post
+import io.ktor.server.routing.get
+import io.ktor.server.routing.RoutingCall
+import io.ktor.server.routing.routing
+import io.ktor.server.sse.SSE
+import kotlinx.coroutines.delay
+import java.time.Instant
+
+/**
+ * Mini server to send server side events over HTTP to facilitate testing.
+ *
+ * @author Francisca Eze
+ */
+object SSETestServer {
+
+    private val server = embeddedServer(Netty, port = 8888) {
```

Comment:
Use a random available port. Use https://docs.oracle.com/javase/7/docs/api/java/net/ServerSocket.html#ServerSocket%28int%29 with arg 0, get the used port and close the ticket to release it.